app/views/listings/_listing.json.jbuilder

```diff
@@ -1,2 +1,2 @@
-json.extract! listing, :id, :title, :description, :type, :created_at, :updated_at
```

Comment:
I think we'll still need a description? (e.g. $100 cash requested, $50 offered). Relatedly, how will we handle an ask or offer that's not already an available tag? 

Also I think we should consider adding to date by when the listing is needed, and date by when it should be removed from the site.  Do you agree?

Reply:
Yep, we'll need one or two more fields but i was thinking they could be more specifically named than a generic `description`. Maybe not, will see.

I'm working on allowing users to add tags from the form.

I like the idea of `needed_by` and `list_until`. Added an issue to make sure we get to that:
https://github.com/maebeale/mutual-aid/issues/20